pt-BR/getting_started.md

```diff
@@ -1883,13 +1882,13 @@ Then you make the `app/views/articles/show.html.erb` look like the following:
 <%= link_to 'Back', articles_path %>
 ```
 
-The second render just defines the partial template we want to render,
-`comments/form`. Rails is smart enough to spot the forward slash in that
-string and realize that you want to render the `_form.html.erb` file in
-the `app/views/comments` directory.
+O segundo render apenas define o template de partial que queremos renderizar,
```

Comment:
```suggestion
O segundo render apenas define o template de *partial* que queremos renderizar,
```
@campuscode/guia-rails-team e render, vamos em itálico sem tradução?

Reply:
Nesse caso sim @erikacamposdesign e @gionaufal. Trata-se de uma referência ao trecho de código da documentção, então não vamos traduzir e exibir em itálico.